app/models/container_quota_item.rb

```diff
@@ -1,3 +1,26 @@
 class ContainerQuotaItem < ApplicationRecord
   belongs_to :container_quota
+
+  virtual_column :quota_desired_display,  :type => :string
+  virtual_column :quota_enforced_display, :type => :string
+  virtual_column :quota_observed_display, :type => :string
+
+  def quota_desired_display
+    quota_display(quota_desired)
+  end
+
+  def quota_enforced_display
+    quota_display(quota_enforced)
+  end
+
+  def quota_observed_display
+    quota_display(quota_observed)
+  end
+
+  private
+
+  def quota_display(quota)
+    val = quota == quota.to_i ? quota.to_i : quota
```

Comment:
@zakiva why do we need this?

Reply:
I want to get rid of the `.0` when displaying integers values in the UI, espically for counts of objects it will be strange to see a count of say `5.0 pods`

Reply:
@zakiva I suppose this is can be achieved in a cleaner way with:

```ruby
def quota_display(quota)
    (quota % 1 == 0) ? quota.to_i.to_s : quota.to_s
end
```

Reply:
nice, I will change that